CHANGELOG.md

```diff
@@ -5,7 +5,7 @@ This CHANGELOG follows the format listed at [Our CHANGELOG Guidelines ](https://
 Which is based on [Keep A Changelog](http://keepachangelog.com/)
 
 ## [Unreleased]
-
+- update cpu and memory checks to follow comparison operator in linux plugins check (greater than or equal to) (@derekgroh)
```

Comment:
It's important to note that this change while unlikely will negatively affect someone some people may be very specific and this breaks their old expectations.
```suggestion
### [Breaking Changes]
- update cpu and memory checks to follow comparison operator in linux plugins check (greater than or equal to) (@derekgroh)
```

Reply:
A noteworthy change, but this is not breaking by definition.

Reply:
It depends on how you define breaking. It absolutely changes the expectation of the outcome what will happen given the same input. Even though its a good call it is absolutely breaking. It does not matter if it affects 1 or millions of users, breaking is breaking. It's the only way semver can be trusted.

Take for example:
```ruby
# use the same value as input
current_value = 50

# testing old behavior would result in an OK
if current_value > 50
  critical
else
  ok
end

# testing new behavior will result in a CRITICAL
if current_value >= 50
  critical
else
  ok
end
```